src/services/conversionRates/coingecko.js

```diff
@@ -20,6 +21,7 @@ const fetchCoingecko = async (timestampMS, coingeckoId, toSymbol) => {
     );
   } catch (e) {
     logger.error(`coingecko fetch (id:${coingeckoId}, toSymbol:${toSymbol})`, e);
+    Sentry.captureMessage(e);
```

Comment:
Should this one and others be replaced by `captureException`?
Also in case `captureError` is not enough informative to show where the exception is happened or its had to find out at first glance (need referring to code), I suggest passing some context like this guide:
https://docs.sentry.io/platforms/node/enriching-events/context/#passing-context-directly

Reply:
I changed it to captureException and now I send the stack trace

Reply:
![Screen Shot 1400-05-04 at 10 29 40](https://user-images.githubusercontent.com/9850545/126949398-bd7b1fe1-dcaf-4deb-a298-c60ccab2ac8f.png)